### ISSUE TITLE
Fastify adapter not triggering middleware when query parameter is present

### ISSUE THREAD

**Issue by josec89** [nestjs/nest#1474]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Middelware is not triggered when using `FastifyAdapter` and the url includes `query` params.

`/dosomething` --> Middleware is called
`/dosomething?` --> Middleware  is **not called**
`/dosomething?pid=1406` --> Middleware  is **not called**

## Expected behavior

Middleware should be called regardless the query parameters involved

## Minimal reproduction of the problem with instructions

1. Use `FastifyAdapter`
2. Create a middleware and a controller
3. Make use of that middleware in all the routes of that controller

```ts
    configure(consumer: MiddlewareConsumer) {
        consumer
            .apply(UserMiddleware)
            .forRoutes(MyController)
      }
```

4. Call one endpoint in that controller with and without query params.

`/dosomething` --> Middleware is called
`/dosomething?` --> Middleware  is **not called**
`/dosomething?pid=1406` --> Middleware  is **not called**

## Environment

<pre><code>
Nest version: 5.6.2

 
For Tooling issues:
- Node version: v10.15.0  
- Platform: Mac  

Others:

</code></pre>

**Comment by Tsury** [comment]
Seems to be related to an issue with Nest and FastifyAdapter in general, not necessarily related to query parameters
https://github.com/nestjs/nest/issues/972#issuecomment-467052317

**Comment by josec89** [comment]
@Tsury I'll try to reproduce the error again with the latest version, hope it's indeed fixed :) 🤞

**Comment by kamilmysliwiec** [comment]
It should be fixed in 5.7.4

**Comment by Suvitruf** [comment]
Have this issue in 6.1.0.
